src/rebar_prv_install_deps.erl

```diff
@@ -367,9 +367,8 @@ handle_upgrade(AppInfo, Profile, SrcDeps, PkgDeps, SrcApps, Level, State, Locks)
                 {true, AppInfo1} ->
                     handle_dep(AppInfo1, Profile, SrcDeps, PkgDeps, SrcApps,
                                Level, State, Locks);
-
                 {false, AppInfo1} ->
-                    {[AppInfo1|SrcDeps], PkgDeps, SrcApps, State, Locks}
+                    {[AppInfo1|SrcDeps], PkgDeps, [AppInfo1|SrcApps], State, Locks}
```

Comment:
Why was this needed again? Any impact we expect on regular apps being installed?